en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Supported modules
+---
+
+# Commercially supported modules for Silverstripe CMS 5
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to semantic version. They are covered by:
+- our [security release process](./Managing_Security_Issues) and 
```

Comment:
```suggestion
- our [security release process](Managing_Security_Issues) and 
```
It's already relative to the current location without using `./`, and we don't tend to use `./` elsewhere in the docs.
This applies to any other links you've done this way in this PR.